README.md

```diff
@@ -8,6 +8,8 @@ AWS related modules and plugins supported by the Ansible community are in the [c
 
 This collection has been tested against following Ansible versions: **>=2.9.10**.
 
+For collections that support Ansible 2.9, please ensure you update your `network_os` to use the 
+fully qualified collection name (for example, `cisco.ios.ios`). 
```

Comment:
But we should probably keep only this?

Reply:
imo, the README.md for  2.x.x releases must reference to `stable`  2 branch. It was probably forgotten in the past releases.

Maybe the best to fix this is, to release a bugfix release 2.2.1

Reply:
Ah, I've misread everything. this is a PR into main.  
So yes, the README should not changes to ` stable-2`  in this case

`collection_prep_add_docs -b main  -p .`